californium-core/src/main/java/org/eclipse/californium/core/network/serialization/Serializer.java

```diff
@@ -34,7 +35,14 @@
  */
 public class Serializer {
```

Comment:
No I don't see a good reason either - it seems to have some convenience wrappers, but nothing CoApEndpoint can't do itself. I was trying to minimize changes outside those needed to support TCP, but happy to nuke this class.